interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,84 @@
+import controller.volunteer_controller as vc
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
+    def __init__(self):
```

Comment:
I think this `__init__` function is not required as its parent class doesn't have one.